cql3/restrictions/multi_column_restriction.hh

```diff
@@ -54,6 +54,17 @@ namespace cql3 {
 
 namespace restrictions {
 
+inline
+expr::tuple_constructor
+column_definitions_as_tuple_constructor(const std::vector<const column_definition*> defs) {
```

Comment:
This passes a vector by value -- is there an ampersand missing in front of `defs`?

Reply:
Yes, fixed.